README.md

```diff
@@ -27,7 +27,9 @@ Connecting to EBB on /dev/tty.usbmodem1461
 To install saxi on a Raspberry Pi, first install node.js if you haven't already:
 
 ```
-$ curl -sL https://deb.nodesource.com/setup_11.x | sudo -E bash -
+$ curl -sL https://deb.nodesource.com/setup_12.x | sudo -E bash -
+$ sudo apt-get install -y nodejs
+$ sudo chmod a+rwx /usr/lib/node_modules
```

Comment:
```suggestion
```
I don't think this is advisable. Here's what npm's documentation has to say about installing modules globally without root: https://docs.npmjs.com/resolving-eacces-permissions-errors-when-installing-packages-globally

Reply:
Haha, yes, I went with this because it was for raspberrypi, wouldn't do that on my computer 🙃